app/models/course/assessment/question/programming.rb

```diff
@@ -1,11 +1,21 @@
 # frozen_string_literal: true
+ActiveRecord::ActsAs::InstanceMethods.class_eval do
+  def dup
```

Comment:
Ok managed to get rid of this by using the same strategy as lesson plan.

Duplicate the `actable` but copy the attributes of the `acts_as` over.